app/src/ui/lib/aria-types.ts

```diff
@@ -0,0 +1,5 @@
+import { HTMLAttributes } from 'react'
+
+export type AriaHasPopupType = HTMLAttributes<
+  string | boolean | undefined
```

Comment:
While it seems to be irrelevant, I don't think this is correct 🤔 AFAIK `HTMLAttributes` expects the type to be an Element (if you dive into `DomAttributes` you will see that).

Anyway, just a very minor thing.